internal/http/services/owncloud/ocs/data/capabilities.go

```diff
@@ -110,6 +118,7 @@ type CapabilitiesDav struct {
 	Trashbin                       string   `json:"trashbin" xml:"trashbin"`
 	Reports                        []string `json:"reports" xml:"reports>element" mapstructure:"reports"`
 	ChunkingParallelUploadDisabled bool     `json:"chunkingParallelUploadDisabled" xml:"chunkingParallelUploadDisabled"`
+	Spaces                         bool     `json:"spaces" xml:"spaces"`
```

Comment:
Why do we need it here under the Core key?

Reply:
Consistency? What are the implications?

Reply:
It is not under the Core object